plots/laws/gravity/throwing_angle.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# Description
+## You throw the grenade to enemies. Which angle of throwing should you choose to hit the farest enemy?
+
+from math import pi
+from sympy import sin, cos
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics import symbols, Eq, pretty, solve, Function
+
+flight_time = symbols('flight_time')
+throwing_velocity = symbols('throwing')
+alpha = symbols('alpha')
+Distance = symbols('Distance')
+velocity_vertical = symbols('v_vertical')
+velocity_horizontal = symbols('v_horizontal')
+velocity_vertical = throwing_velocity * sin(alpha)
+velocity_horizontal = throwing_velocity * cos(alpha)
```

Comment:
As we decided to not implement corresponding laws in this PR, add TODOs on what should be extracted to separate laws.